config-templates/salesforce/leads.yaml

```diff
@@ -0,0 +1,71 @@
+data_source:
+  name: salesforce_lead
+  owners:
+    - support@transformdata.io
+  sql_table: salesforce.lead ### Update to your leads schema.table
+  mutability:
```

Comment:
Mutability will be optional in MetricFlow, but we should probably not include this since everyone's sync schedules will differ.